therr-services/maps-service/src/handlers/spaceMetrics.ts

```diff
@@ -124,12 +124,22 @@ const getSpaceMetrics = (req, res) => {
                 startDate,
                 endDate,
             } = req.query;
-            return getFormattedMetrics(startDate, endDate, spaceId).then(([currentMetrics, previousMetrics]) => res.status(200).send({
+            return getFormattedMetrics(startDate, endDate, spaceId).then(([currProspectMetrics, prevProspectMetrics, currImpressionMetrics, prevImpressionMetrics, currVisitMetrics, prevVisitMetrics]) => res.status(200).send({
                 space,
-                metrics: currentMetrics,
+                metrics: {
+                    "space.user.prospects": currProspectMetrics,
+                    "space.user.impressions": currImpressionMetrics,
+                    "space.user.visits": currVisitMetrics
+                },
                 aggregations: {
-                    metrics: aggregateMetrics(currentMetrics),
-                    previousSeriesPct: getPercentageChange(currentMetrics, previousMetrics),
+                    metrics: {
+                        "space.user.prospects": aggregateMetrics(currProspectMetrics),
+                        "space.user.impressions": aggregateMetrics(currImpressionMetrics),
+                        "space.user.visits": aggregateMetrics(currVisitMetrics)
+                    },
+                    previousSeriesAPct: getPercentageChange(currProspectMetrics, prevProspectMetrics),
+                    previousSeriesBPct: getPercentageChange(currImpressionMetrics, prevImpressionMetrics),
+                    previousSeriesCPct: getPercentageChange(currVisitMetrics, prevVisitMetrics)
```

Comment:
Can you think of a better response body that is more dynamic? Maybe something like... 
```
{
  aggregations: {
      "space.user.prospects": {
        metrics: aggregateMetrics(currProspectMetrics),
        previousSeriesPct: getPercentageChange(currProspectMetrics, prevProspectMetrics),
      },
      "space.user.impressions": {
        metrics: aggregateMetrics(currImpressionMetrics),
        previousSeriesPct: getPercentageChange(currImpressionMetrics, prevImpressionMetrics),
      },
      "space.user.visits": {
        metrics: aggregateMetrics(currVisitMetrics),
        previousSeriesPct: getPercentageChange(currVisitMetrics, prevVisitMetrics),
      }
  }
}
```

We may want to allow the user to request specific metrics by name in the future